### DISCUSSION TITLE
Demo Firefly site Personal Access Token

### DISCUSSION THREAD

**Discussion by fauxir** [firefly-iii/firefly-iii#8292] [Firefly III]
Hi, I'm trying to use the Demo FireFly Personal Access Token to test some features of my email monthly report app: https://github.com/fauxir/firefly-iii-monthly-email-report

Compared to the regular instance that I have this one won't work for some reason as when I try go get the categories it will throw an error (basically at the first API call). Is there anything that I need to do in order to make it work or the way the demo instance was design is creating this error for safety reasons. TIA!

**Comment by JC5** [comment]
The demo site doesn't accept most standard user agents like "curl" or "python", maybe that's the problem?
